cmd/config_dirigent_dandelion.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Add suffix to filename `_rps`. And have one that is not RPS. Have a look what we have in the folder.